addons/l10n_ch/models/res_bank.py

```diff
@@ -272,7 +272,8 @@ def _is_qr_iban(self):
         QR-codes. They are formed like regular IBANs, but are actually something
         different.
         """
-        self.ensure_one()
+        if not self:
+            return UserError(_("Bank account is mandatory to generate QR invoice"))
```

Comment:
This could be a bit more detailed, for example, let the user know where to set up the bank account maybe.

Reply:
right on the invoice, I don't know how to do this in more detail as the message raised on the invoice and if you close the message window, the bank account field right after under it